eventActions/cotwActions.js

```diff
@@ -81,6 +85,8 @@ class cotwActions {
 						return challengeName.replace('Challenge of the Week: ', '');
 					});
 				store.set('challengeName', challengeName);
+				const emote = config.emotes.cotwReflection
```

Comment:
```suggestion
				const emote = config.emotes.cotwReflection;
```

To make the linter happy